File: 08-1-1-Observer/ConsoleView.cs
namespace Observer
{
    using System;

    public sealed class ConsoleView : IView
    {
        private readonly int _id;

        public ConsoleView(int id)
        {
            _id = id;
        }

        public void Update(float temperature, float humidity, float windSpeed)
        {
            Console.WriteLine($"View #{_id} | Temperature: {temperature} C° | Humidity: {humidity} % | Windspeed: {windSpeed} m/s");
        }
    }
}

File: 08-DesignPatterns-Java/Starter/Observer/src/ConsoleView.java
public class ConsoleView implements IView {
    private final int _id;

    public ConsoleView(int id) {
        _id = id;
    }

    public void update(float temperature, float humidity, float windSpeed) {
        System.out.println("View #" + _id + " | Temperature: " + temperature + " C° | Humidity: " + humidity + " % | Windspeed: " + windSpeed + " m/s");
    }
}